bitrise.yml

```diff
@@ -98,3 +89,13 @@ workflows:
             bitrise share start -c "${MY_STEPLIB_REPO_FORK_GIT_URL}"
             bitrise share create --stepid "${BITRISE_STEP_ID}" --tag "${BITRISE_STEP_VERSION}" --git "${BITRISE_STEP_GIT_CLONE_URL}"
             bitrise share finish
+  test:
+    steps:
+    - script:
+        title: go test
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+
+            go test ./...
```

Comment:
definitely! we have a template for Go tools (https://github.com/bitrise-io/bitrise/blob/master/bitrise.yml#L87)